zio-k8s-client/src/main/scala/com/coralogix/zio/k8s/client/config/KeyManagers.scala

```diff
@@ -22,7 +22,7 @@ private object KeyManagers {
       _                   <-
         propertyKeyStoreFile match {
           case Some(file) =>
-            ZManaged.fromAutoCloseable(ZIO.attempt(new FileInputStream(file))).use { stream =>
+            ZIO.fromAutoCloseable(ZIO.attempt(new FileInputStream(file))) flatMap { stream =>
```

Comment:
The equivalent of `.use` is `ZIO.scoped { managed.flatMap { ... } }```. So you should introduce a new scope here that closes the file at the same place as the `use` ended before.

Reply:
done.